pkg/functions/functions.go

```diff
@@ -86,3 +95,8 @@ func (f Functions) Select(name string) Functions {
 
 	return funcs
 }
+
+func jsonString(v interface{}) string {
+	b, _ := json.Marshal(v)
```

Comment:
Is it a good idea to just discard errors here? These functions eventually operate on unconstrained user input... so I'd like to be robust against any cycles or worse.